src/App.vue

```diff
@@ -361,7 +362,7 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
   readonly window = window
 
   /** The Update Current JS Date timer id. */
-  private updateCurrentJsDateId = 0
+  private updateCurrentJsDateId = null as any // NodeJS.Timeout
```

Comment:
Node complains that this variable is not a number (though I think it is a number in the browser) so I just typed it as any.